src/Compilers/Core/Portable/StrongName/DesktopStrongNameProvider.cs

```diff
@@ -213,7 +213,7 @@ internal override StrongNameKeys CreateKeys(string keyFilePath, string keyContai
                     var fileContent = ImmutableArray.Create(ReadAllBytes(resolvedKeyFile));
```

Comment:
Normally I agree, but ReadAllBytes is publicly accessible and overridable. I want to guard against any exceptions from `ReadAllBytes`, not just our implementation. I also think the implicit contract, based on our current implementation, is that it's OK for that method to throw.